angular-chart.js

```diff
@@ -321,12 +325,16 @@
       });
     }
 
-    function setLegend (elem, chart) {
+    function setLegend (elem, chart, scope) {
       var $parent = elem.parent(),
-          $oldLegend = $parent.find('chart-legend'),
-          legend = '<chart-legend>' + chart.generateLegend() + '</chart-legend>';
-      if ($oldLegend.length) $oldLegend.replaceWith(legend);
-      else $parent.append(legend);
+          $oldLegend = $parent.find('chart-legend');
+      if (scope.legend) {
```

Comment:
Missing `&& scope.legend !== 'false'`. If user puts `false` in the markup, it is interpreted as a string hence `scope.legend` will be `true`. See #23. Usually not a great idea to remove code if you don't know why it's here in the first place.